packages/larva-tokens/src/brands/wwd.json

```diff
@@ -1,23 +1,46 @@
 {
 	"imports": [
-		"../base/all.json"
+		"../base/aliases.json",
+		"../base/color.json",
+		"../base/font-family.json",
+		"../base/background-color.json",
 	],
 	"props": {
 		"BACKGROUND_COLOR_BRAND_PRIMARY": {
 			"value": "{!RED_45}",
 			"type": "color",
 			"category": "background-color"
 		},
+		"BACKGROUND_COLOR_BRAND_SECONDARY": {
+			"value": "{!BLACK}",
+			"type": "color",
+			"category": "background-color"
+		},
+		"BACKGROUND_COLOR_BRAND_ACCENT": {
+			"value": "{!WHITE_95}",
+			"type": "color",
+			"category": "background-color"
+		},
 		"BORDER_COLOR_BRAND_PRIMARY": {
 			"value": "{!RED_45}",
 			"type": "color",
 			"category": "hr-color"
 		},
+		"BORDER_COLOR_BRAND_SECONDARY": {
+			"value": "{!BLACK}",
+			"type": "color",
+			"category": "hr-color"
+		},
 		"COLOR_BRAND_PRIMARY": {
 			"value": "{!RED_45}",
 			"type": "color",
 			"category": "text-color"
 		},
+		"COLOR_BRAND_SECONDARY": {
```

Comment:
I think you only need to add new values here if you are overriding what comes from default - the idea is that by importing `all` above, you will have default definitions for everything and do not need to add them.

Reply:
ah missed this, makes sense, fixing!